Batteries/Data/List/OfFn.lean

```diff
@@ -0,0 +1,49 @@
+/-
+Copyright (c) 2024 Lean FRO. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kim Morrison
+-/
+import Batteries.Data.List.Lemmas
+import Batteries.Data.Fin.Lemmas
+
+/-!
+# Theorems about `List.ofFn`
+-/
+
+namespace List
+
+@[simp]
+theorem length_ofFn (f : Fin n → α) : (ofFn f).length = n := by
+  simp only [ofFn]
+  induction n with
+  | zero => simp
+  | succ n ih => simp [Fin.foldr_succ, ih]
+
+@[simp]
+theorem getElem_ofFn (f : Fin n → α) (i : Nat) (h : i < (ofFn f).length) :
```

Comment:
It doesn't look like this is a concern currently in the code base but should this and `getElem?_ofFn` be `protected`?

Reply:
Sure. Better done early.